SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1750,15 +1759,22 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 		const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
 
 		Connection->Advance();
+
```

Comment:
Want this?

Reply:
I think it's worth keeping as these seem like distinct stages to me. I don't feel strongly though, so can remove

Reply:
No I don't care either tbh :p